.github/workflows/ci.yml

```
@@ -141,12 +147,15 @@ jobs:
         cat /proc/cpuinfo
         ./util/opensslwrap.sh version -c
     - name: make test
-      run: .github/workflows/make-test
+      run: .github/workflows/make-test || touch .test_failure
     - name: save artifacts
```

Could you please try this instead?

```suggestion
      run: .github/workflows/make-test
    - name: save artifacts
      if: failure()
```

---

Now it looks, the artifiacts are only saved when an error happens, which is also not what I want.

---

Hmm, I never had the impression that it is too onerous to have those artifacts even if the test was passed.
Since it may be of interest to inspect the artifacts generated by certain targets or from a specific pull request,
even when no error is detected.

---

Hmm, I can't really think of such case. However `if: success() || failure()` would run it both in case of success and failure.

---

ok, let's try it this way

---

Can you please push the change?

---

Yes, I am currently testing it.

---

Ok test looks good now.
This was indeed a nice improvement, thanks for the suggestion...